starboard/tools/api_leak_detector/api_leak_detector.py

```diff
@@ -94,6 +94,11 @@
     'realloc',
     'strcasecmp',
     'strncasecmp',
+    'vsnprintf',
```

Comment:
Why do you add the symbols here if they're already added in c99.md? They shouldn't be necessary in both places

Reply:
Good catch!